qa/automation/CareerPortal.Tests.Scenarios/4. Sending CV/4.1.Sending CV.feature

```diff
@@ -0,0 +1,132 @@
+Feature: 4.1. Sending CV
+
```

Comment:
I would be add "Background" with Given which described that user should opened https://career.quantori.com/ru
And scrolled down to the CV form